velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -239,7 +239,14 @@ std::vector<std::string> AggregationFuzzerBase::generateSortingKeys(
     std::vector<std::string>& names,
     std::vector<TypePtr>& types) {
   std::vector<std::string> keys;
-  auto numKeys = boost::random::uniform_int_distribution<uint32_t>(1, 5)(rng_);
+  uint32_t numKeys;
+  // 50% chance that there will only be 1 order by key so RANGE frames or ROWS
```

Comment:
This is a bit unclear, maybe modify it to `50% chance that there will only be 1 order by key so k-RANGE frames can be generated` ?